tests/test_repositoryCfg.py

```diff
@@ -157,37 +160,36 @@ def testStorageRepoCfgCache(self):
         """Tests that when a cfg is gotten from storage it is cached."""
         butler = dp.Butler(outputs=dp.RepositoryArgs(mode='w',
                                                      mapper=dpTest.EmptyTestMapper,
-                                                     root=os.path.join(ROOT, 'repositoryCfg/a')))
+                                                     root=os.path.join(self.testDir, 'a')))
         del butler
         storage = dp.Storage()
         self.assertEqual(0, len(storage.repositoryCfgs))
-        cfg = storage.getRepositoryCfg(os.path.join(ROOT, 'repositoryCfg/a'))
-        self.assertEqual(cfg, storage.repositoryCfgs[os.path.join(ROOT, 'repositoryCfg/a')])
+        cfg = storage.getRepositoryCfg(os.path.join(self.testDir, 'a'))
+        self.assertEqual(cfg, storage.repositoryCfgs[os.path.join(self.testDir, 'a')])
 
 class TestNestedCfg(unittest.TestCase):
 
-    rootDir = os.path.join(ROOT, 'repositoryCfg_TestNestedCfg')
-
     def setUp(self):
-        self.tearDown()
+        self.testDir = tempfile.mkdtemp(dir=ROOT,
+                                        prefix="repositoryCfg_TestNestedCfg-")
```

Comment:
I see you just used what was already there but the prefix here can probably just be TestNestedCfg